src/kubernetes/kubernetes.go

```diff
@@ -8,10 +8,13 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/util/yaml"
 	"k8s.io/client-go/rest"
-	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	ctrl "sigs.k8s.io/controller-runtime"
 )
 
+const clusterFile = "CLUSTER_CONFIG"
+
 type Cluster struct {
 	client.Client
 	Name  string
```

Comment:
(Line 23)
The `ioutil.ReadFile` is deprecated and should be replaced with `os.ReadFile`

Reply:
You are right, it's calling os.ReadFile underneath.